lib/addTemplatesToModule.js

```diff
@@ -77,7 +77,8 @@ function constructTemplateNode(t, node, isRoot) {
 				}
 				if (attrKeys.length > 0) {
 					template.push(t.ObjectProperty(t.identifier('attrs'), t.ObjectExpression(attrKeys.map(function (attrKey) {
-						return t.ObjectProperty(t.identifier(attrKey), getObjectValue(t, attrs[attrKey]));
+						var id = t.identifier("'" + attrKey + "'");
```

Comment:
Check that you need to add quotes first before you do? Minor, but you could save literally ones of precious, precious bytes.